### ISSUE TITLE
BUG: Checks not passing value properly

### ISSUE THREAD

**Issue by ghosty212** [wh1te909/tacticalrmm#568]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: chrome
 - RMM Version (as shown in top left of web UI): 0.6.13

**Installation Method:**
  - [X ] Standard

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): 1.5.7
- Agent OS: Windows 10 Pro, 64 bit v2009 (build 19042.1052)

**Describe the bug**
Script Variables cannot be used in script checks. Using {{agent.name}} or {{site.name}} in a script check leads to the variable being {agent.name} or {site.name} instead of the actual agent.name or site.name. Same for custom fields.

**To Reproduce**
1. 
```
# Test-CustomField.ps1

param (
   [string] $ClientName,
)

Write-Output "Client: $ClientName"
```
2. add Test-CustomField.ps1 as script check with variable ( -ClientName {{client.name}} )
3. run check
4. see that output is: `Client: {client.name}`

**Expected behavior**
Output should be the actual value of client.name. Like it is when running the script via "run script" or as automated task.

**Additional context**
there was a similiar bug with automated tasks before and fixed in #514

**Comment by wh1te909** [comment]
thanks for finding this, fixed and will be in next release

**Comment by wh1te909** [comment]
released in 0.6.15

**Comment by cm-slu** [comment]
I just tested it works.
Thanks the team.
